cylc/flow/cfgspec/workflow.py

```diff
@@ -811,7 +811,7 @@ def get_script_common_text(this: str, example: Optional[str] = None):
             This setting can be overridden by the reserved keyword argument
             ``sequential`` in individual xtrigger declarations.
 
-            One sequential xtrigger on a parentless task with multiple
+            One sequential xtrigger on a parent-less task with multiple
```

Comment:
Are we doing this for the spell-checker?  If so, can we just add "parentless" to the dictionary?  I don't think "parent-less" is right.

Reply:
Yeah, just for spell checker .. we did this to the other one too .. ok will do